### PR TITLE
fix(*): Add dns_servers as configuration option

### PR DESCRIPTION
**Pull request by Moep90**
## what
* Describe high-level what changed as a result of these commits (i.e. in plain-english, what do these changes mean?)
* Use bullet points to be concise and to the point.

## why
* Provide the justifications for the changes (e.g. business case).
* Describe why these changes were made (e.g. why do these commits fix the problem?)
* Use bullet points to be concise and to the point.

## references
* Link to any supporting jira issues or helpful documentation to add some context (e.g. stackoverflow).
* Use `closes #123`, if this PR closes a Jira issue `#123`

### CODE REVIEW

File: versions.tf [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Terraform version
 terraform {
-  required_version = ">= 1.6.6"
+  required_version = ">= 1.5.7"
```
[the remark this answers is not in the document]
Comment (rhhammond) [reply versions.tf:3]: Maybe this is related to the Hashicorp BSL announcement last year?

[see item 7 on the FAQ](https://www.hashicorp.com/license-faq)

I am interested in using this change in my project - what needs to happen to promote it?

[the remark this answers is not in the document]
Comment (nileshgadgi) [reply versions.tf:3]: 1.5.7 or above version will also support BSL, so I guess no need to downgrade the terraform version.
https://www.hashicorp.com/license-faq#products-covered-by-bsl
@rhhammond @Moep90

**Review by nileshgadgi — CHANGES_REQUESTED** [review]
upgrade the terraform version back to 1.6.6.

**Review by anmolnagpal — APPROVED** [review]

File: versions.tf (hunk 1 above)
[the remark this answers is not in the document]
Comment (nileshgadgi) [reply versions.tf:3]: we sholud 1.5.7 version to enable support BSL

**Review by nileshgadgi — APPROVED** [review]
LGTM!

**Pull request merged**